JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -46,7 +63,83 @@ public class OAuthSampleApp implements Callable<Integer> {
 
     @Option(names = { "-s", "--client-secret" }, description = "Client Secret")
     private String clientSecret = "";
+	
+	private static final String OAUTH_CLIENT_ID = System.getenv("OAUTH_CLIENT_ID");
+    private static final String OAUTH_REALM = System.getenv("OAUTH_REALM");
+    private static final String OAUTH_TOKEN_USER = System.getenv("OAUTH_TOKEN_USER");
+    private static final String OAUTH_TOKEN_PASSWORD = System.getenv("OAUTH_TOKEN_PASSWORD");
+
+    private static String OAUTH_SERVER = System.getenv("OAUTH_SERVER"); 
+    private static String OAUTH_HTTP_PORT =  System.getenv("OAUTH_HTTP_PORT");
+
+    private static String getTokenUrl()
```

Comment:
getTokenUrl, getHttpsTokenUrl, getIssuerUri, and getDiscoveryUrl are all unneeded. The only ones being used is the token/https token url and if we just have that be provided through the new environment variable then we don't need the get function

Reply:
These unneccessary methods are removed